### PR TITLE
Update 2020-06-03-dalalyan20a.md

### PR DESCRIPTION
**Pull request by marcj75**
Changed author order

### CODE REVIEW

File: _posts/2020-06-03-dalalyan20a.md [hunk 1]
```diff
@@ -23,12 +23,12 @@ lastpage: 1341
 page: 1331-1341
 order: 1331
 cycles: false
-bibtex_author: Dalalyan, Arnak and Schreuder, Nicolas and Brunel, Victor-Emmanuel
+bibtex_author: Schreuder, Nicolas and Dalalyan, Arnak and Brunel, Victor-Emmanuel
```
Comment (lawrennd) [inline _posts/2020-06-03-dalalyan20a.md]: This order doesn't match the PDF ... where Dalalyan seems to be the last author.

File: _posts/2020-06-03-dalalyan20a.md [hunk 2]
```diff
@@ -23,12 +23,12 @@ lastpage: 1341
...
 author:
-- given: Arnak
-  family: Dalalyan
 - given: Nicolas
   family: Schreuder
+- given: Arnak
```
Comment (lawrennd) [inline _posts/2020-06-03-dalalyan20a.md]: This order doesn't match the PDF ... where Dalalyan seems to be the last author.

**Review by lawrennd — CHANGES_REQUESTED** [review]
I checked the PDF, and the new author order still doesn't match.

**Comment by marcj75** [comment]
Dear Neil,

you're great!
The authors have sent me the wrong order :(
Now, it is correct.

Thank you,
Marcello

File: _posts/2020-06-03-dalalyan20a.md [hunk 3]
```diff
@@ -23,14 +23,14 @@ lastpage: 1341
 page: 1331-1341
 order: 1331
 cycles: false
-bibtex_author: Dalalyan, Arnak and Schreuder, Nicolas and Brunel, Victor-Emmanuel
+bibtex_author: Schreuder, Nicolas and Brunel, Victor-Emmanueland Dalalyan, Arnak
```
Comment (lawrennd) [inline _posts/2020-06-03-dalalyan20a.md]: ``
```suggestion
bibtex_author: Schreuder, Nicolas and Brunel, Victor-Emmanuel and Dalalyan, Arnak 
```

**Review by lawrennd — COMMENTED** [review]
Just adding space in.

**Pull request merged**